docker/candlepin-base-cs8/setup-devel-env.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+#
+# Sets a system up for a candlepin development environment (minus a db,
+# handled separately), and an initial clone of candlepin.
+
+set -ve
+
+source /root/dockerlib.sh
+
+export JAVA_VERSION=1.8.0
+export JAVA_HOME=/usr/lib/jvm/java-$JAVA_VERSION
+
+# Install & configure dev environment
+yum install -y epel-release
+
+PACKAGES=(
+    gcc
```

Comment:
I've posted PR https://github.com/candlepin/candlepin/pull/3447 because I realized we have too many dependencies here that we don't need. You can remove all the dependencies I removed from the centos7 script here, except from the `postgresql` (because we need it in CS8 instead of the rh-postgresql96/12 to get access to the pg_isready command)

Reply:
Updated

Reply:
I removed postgresql-jdbc (but kept postgresql) from the C7 image. Is it needed in CS8 for some other reason? (if not, then let's just remove it)

Reply:
Good catch, yeah I missed postgresql-jdbc. Removed it!